xbmc/cores/VideoPlayer/VideoReferenceClock.cpp

```diff
@@ -204,7 +204,7 @@ void CVideoReferenceClock::SetSpeed(double Speed)
     if (Speed != m_ClockSpeed)
     {
       m_ClockSpeed = Speed;
-      CLog::Log(LOGDEBUG, "CVideoReferenceClock: Clock speed %f%%", m_ClockSpeed * 100.0);
+      CLog::Log(LOGDEBUG, "CVideoReferenceClock: Clock speed %0.2f %%", m_ClockSpeed * 100.0);
```

Comment:
Please remove the 0. or we will have crashes on platforms with new enough libfmt: https://github.com/xbmc/xbmc/pull/15224

Reply:
False alert! Sorry